src/Powercord/plugins/pc-commands/commands/disable.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    command: 'disable',
+    description: 'Disable a plugin',
+    usage: '{c} [ Plugin Name ]',
+    aliases: [],
+    func: (args) => {
+        let result;
+        if(powercord.pluginManager.plugins.has(args[0])) {
+            if(args[0].toLowerCase() === 'pc-commands') {
+                result = 'You cannot disable that!'
+            } else if(!powercord.pluginManager.isEnabled(args[0])) {
+                result = `\`${args[0]}\` is already disabled`
+            } else {
+            result = `\`${args[0]}\` has been disabled`
+            powercord.pluginManager.disable(args[0])
+        }} else {
+            result = 'You do not have that plugin installed'
+        }
+
+        return { 
+            send: false,
+            result
+        }
+    }
```

Comment:
As already brought up in the '#powercord-core' channel of Powercord's Discord, you should consider putting the autocomplete function, `autocompleteFunc()`, to good use to allow for additional functionality as well as error prevention.

I've gone out of my way and have already made a start on this implementation, however you'll need to remove the condition that checks if the arguments are equal to "pc-commands" as the autocomplete menu now takes care of this.

I'd also kindly suggest that a similar method is put under your "[enable.js](https://github.com/powercord-org/powercord/pull/224/files#diff-264f847bb2bb39fb43385a996a88d46b)" file for the sake of consistency.

```suggestion
    },

    autocompleteFunc (args) {
        const plugins = powercord.pluginManager.getPlugins()
            .sort((a, b) => a < b ? -1 : 1 || 0)
            .map(plugin => powercord.pluginManager.plugins.get(plugin));

        if (args.length > 1) {
            return false;
        }

        return {
            commands: plugins
                .filter(plugin => plugin.pluginID !== 'pc-commands' &&
                    plugin.pluginID.includes(args[0].toLowerCase()))
                .map(plugin => ({
                    command: plugin.pluginID,
                    description: plugin.manifest.description
                }))
                .slice(0, 10),
            header: 'powercord plugin list'
        };
    }
```